webserver/__init__.py

```diff
@@ -155,21 +155,25 @@ def register_ui(app):
         from webserver.views.login import login_bp
         from webserver.views.user import user_bp
         from webserver.views.datasets import datasets_bp
+        from webserver.views.similarity import similarity_bp
         app.register_blueprint(index_bp)
         app.register_blueprint(data_bp)
         app.register_blueprint(stats_bp)
         app.register_blueprint(login_bp, url_prefix='/login')
         app.register_blueprint(user_bp)
         app.register_blueprint(datasets_bp, url_prefix='/datasets')
+        app.register_blueprint(similarity_bp, url_prefix='/similarity')
```

Comment:
we should be able to disable user-facing parts of this project (similarity pages, eval) even if it's in master until we're ready for other people to use it. Use a config item like `FEATURE_EVAL_LOCATION`